### ISSUE TITLE
Decrease the number of digits after the decimal point

### ISSUE THREAD

**Issue by artenax** [prometheus-community/pro-bing#25]
I noticed that it depends on the system. Linux has 6 digits (e.g. 120.125368 ms), MacOS has 3 digits. In my opinion this is excessive precision. Could you round it down to one digit?

**Comment by SuperQ** [comment]
No, this is up how accurate each measurement is. Having microsecond accuracy is not unusual. It has no impact on float64 precision.

**Comment by artenax** [comment]
I am not a programmer and have little understanding of what you are talking about. I am looking for a replacement for the default ping on Linux for this reason:
"Why does ping run forever by default in Linux?"
https://unix.stackexchange.com/questions/255405/why-does-ping-run-forever-by-default-in-linux
(although I already found a solution with alias ping=ping\ -c4, but still)
I compiled your application, put it in the PATH and edited the source code (ping.go) beforehand to have 4 requests by default and a timeout of 20 seconds.
Clarification: exactly the default. I know there are options like -c, but I'm too lazy to write them.
So, I got almost a standard ping, except the time is too accurate, which is unpleasant to the eye. See the screenshot:

![ping](https://user-images.githubusercontent.com/107228652/224467990-af10cc2d-62ae-4dcf-9ad5-2709b8e510f4.png)
You don't want to change the accuracy of value in the upstream, ok. But tell me how I can fix the source code (ping.go I guess, or golang dependency files?) so that MY local build is the same as the standard ping. Or a little more accurate, but not 6 digits. For example, 1 digit after the decimal point.
I compiled go-ping project binary with go build and just added it to the PATH, to replace the standard ping.
Understand that I am not a programmer, but I know how to compile. However, I do make minimal edits to the source code. I need to fix my build of your project.
I have small requirements for ping (just occasionally check availability of sites, ip addresses), I do not write my projects using go-ping.
